backend/src/main.rs

```diff
@@ -117,3 +106,20 @@ fn load_config_and_init_logger(args: &args::Shared) -> Result<Config> {
 
     Ok(config)
 }
+
+pub fn version() -> String {
+    mod build_info {
+        include!(concat!(env!("OUT_DIR"), "/built.rs"));
+    }
```

Comment:
Huff, I didn't know that's possible! Do we really want that? Is the module visible outside of this function? So many questions. Intuitively I would say "please don't declare mod in function"...

Reply:
It doesn't seem visible, so it's fine, I guess? :shrug: